specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json

```diff
@@ -156,6 +156,76 @@
           }
         ]
       }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers/{customerName}/providers/Microsoft.Subscription/createSubscription": {
+      "post": {
```

Comment:
@jetts is there any reason this is a POST instead of a PUT. Generally, create operations must be PUTs

Reply:
Got it, so a "subscription" isn't being modeled as a true resource - this is just an action. Sounds like this is a deliberate design decision, just wanted to check if there has been any discussion about modelling a subscription as a resource?

Reply:
@antmarti-microsoft There has been talk. We are looking into moving to PUT operations for a new API version. It would require reworking of a lot of the existing APIs to ensure that the new version has the right experience we're shooting for.

Reply:
@jetts if you agree on the ARM recommendations on creating subscription as a 'true' resource for new API version.
Can you capture this decision in a separate file called "feedback.md" and add link to this PR and anything that will help the author to model the spec appropriately in your next iteration/API.

Such recommendations are easy to be lost in conversation and are never implemented.